### DISCUSSION TITLE
Default TopsApp Output Products on cross-polarization or co-polarization?

### DISCUSSION THREAD

**Discussion by cmarshak** [isce-framework/isce2#496] [Q&A] [answered]
What is the default polarization that is used for topsapp without [this parameter](https://github.com/isce-framework/isce2/discussions/495) in the xml file? I cannot find this in the documentation. For example if I used Sentinel-1 SLC's tagged with `SDV` token as done in this [notebook](https://github.com/isce-framework/isce2-docs/blob/master/Notebooks/UNAVCO_2020/TOPS/topsApp.ipynb) that has (VV and VH), what polarization is used? Or am I totally misunderstanding the final products?

**Comment by hfattahi** [comment, marked as answer]
The co-pol VV is used by default.

**Reply by cmarshak** [reply]
So if use HH+HV, then it will be HH as well? only co-pol
